### PR TITLE
Update 07_pytorch_experiment_tracking.ipynb removed tensorboard.dev related information because of shut down.

### PR DESCRIPTION
**Pull request by pritesh2000**
tensorboard.dev has been shut down as of January 1, 2024. so removed related text and links.

amd -> and

Now we've got a pretrained model let's turn into a feature extractor model. ->
Now we've got a pretrained model let's turn it into a feature extractor model.

print(f"Number of batches of size {BATCH_SIZE} in testing data: {len(train_dataloader_10_percent)} (all experiments will use the same test set)")----->
print(f"Number of batches of size {BATCH_SIZE} in testing data: {len(test_dataloader)} (all experiments will use the same test set)")

### CODE REVIEW

**Review by mrdbourke — COMMENTED** [review]
Hi @pritesh2000 , 

I can't seem to view your notebook on GitHub, may I ask how you were submitting it? 

If so, can you show me the code you changed? 

Thank you for the PR.

**Comment by pritesh2000** [comment]
> Hi @pritesh2000 ,
> 
> I can't seem to view your notebook on GitHub, may I ask how you were submitting it?
> 
> If so, can you show me the code you changed?
> 
> Thank you for the PR.

Sorry, that it didn't work.
I forked this repo and changed some files directly from github and then Created PR.
Like this 

image

I am actually creating new PR.
I deleted previous fork and created a new fork.
Cloned forked repository.
Changed both 06_pytorch_transfer_learning.ipynb and 07_pytorch_experiment_tracking.ipynb from my PC using VS code.

AS for the changed code I mentioned changes here:

------------------------------------------------

07_pytorch_experiment_tracking.ipynb 

tensorboard.dev has been shut down as of January 1, 2024. so removed related text and links.

amd -> and

Now we've got a pretrained model let's turn into a feature extractor model. ->
Now we've got a pretrained model let's turn it into a feature extractor model.

`print(f"Number of batches of size {BATCH_SIZE} in testing data: {len(train_dataloader_10_percent)} (all experiments will use the same test set)")`
----->
`print(f"Number of batches of size {BATCH_SIZE} in testing data: {len(test_dataloader)} (all experiments will use the same test set)")`

-----------------------------------------

06_pytorch_transfer_learning.ipynb 

changes of that are like this:

preidctions  -> predictions
fine-tine  -> fine-tune
^some -> some
differnt -> different
assosciated -> associated
populuation -> population
outisde -> outside

Understanding this **performance vs. speed vs. size tradeoff** will come with time and practice. ->
Understanding this **performance vs. speed vs. size** tradeoff will come with time and practice.

Transfer learning often allows to you get good results with a relatively small amount of custom data. ->
Transfer learning often allows you to get good results with a relatively small amount of custom data.

What changes to our code would need to made to use the new API? ->
What changes to our code would need to be made to use the new API?
